src/docker-outside-of-docker/install.sh

```diff
@@ -17,6 +17,7 @@ SOURCE_SOCKET="${SOURCE_SOCKET:-"/var/run/docker-host.sock"}"
 TARGET_SOCKET="${TARGET_SOCKET:-"/var/run/docker.sock"}"
 USERNAME="${USERNAME:-"${_REMOTE_USER:-"automatic"}"}"
 INSTALL_DOCKER_BUILDX="${INSTALLDOCKERBUILDX:-"true"}"
+INSTALL_DOCKER_COMPOSE_SWITCH="${INSTALLDOCKERCOMPOSESWITCH:-"true"}"
```

Comment:
Shouldn't you make changes to `src/docker-outside-of-docker/devcontainer-feature.json` and publicize this Feature option?

Reply:
Use https://github.com/devcontainers/features/blob/main/src/docker-in-docker/devcontainer-feature.json#L54-L58 for reference

Reply:
yes forgot, made changes